nex.go

```diff
@@ -650,10 +662,7 @@ func gen(out *bufio.Writer, x *rule) {
 		out.WriteString("func(r rune) int {\n")
 		var runeCases, classCases string
 		var wildDest int
-		var list edges
-		list = append(list, v.e...)
-		sort.Sort(list)
-		for _, e := range list {
```

Comment:
basically this change is just making `v.e` always be sorted, instead of fixing this one specific iteration.  There's several other iterations over `v.e` in the code.